controllers/policyset/policyset_controller.go

```diff
@@ -337,6 +339,11 @@ func complianceInRelevantClusters(status []*policyv1.CompliancePerClusterStatus,
 			if string(status[i].ComplianceState) == "NonCompliant" {
 				compliance = "NonCompliant"
 				complianceFound = true
+			} else if string(status[i].ComplianceState) == "Pending" {
```

Comment:
Optional: Rather than convert to a string, I think you could use the `Pending` constant from the local `api/v1` package.